src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -0,0 +1,254 @@
+                                <p class="slds-breadcrumb__item slds-line-height_reset">{!v.templateMetadata.labels.objectLabel}</p>
+                                <h1 class="slds-page-header__title slds-m-right_small slds-truncate">
+                                    {!if(v.isReadOnly, v.templateInfo.name, v.templateMetadata.labels.batchTemplateHeader)}
+                                </h1>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="slds-col slds-no-flex slds-grid slds-align-top slds-p-bottom_xx-small headerButtonActions">
+                        <aura:if isTrue="{!v.isReadOnly}">
+                            <lightning:buttonGroup>
+                                <lightning:button label="{!$Label.c.stgBtnEdit}" onclick="{!c.changeModeToEdit}"/>
+                                <lightning:button aura:id="backToTemplates" label="{!v.templateMetadata.labels.backToTemplates}" onclick="{!c.handleButtonClick}"/>
+                            </lightning:buttonGroup>
+                        </aura:if>
+                    </div>
+                </div>
+            </div>
+        </header>
+    </aura:if>
+
+    <!--Body-->
+    <div aura:id="body" class="{!if(v.templateMetadata.labels.sObjectNameNoNamespace == 'Batch_Template__c', 'slds-tabs_card', '')}">
+        <lightning:layout horizontalAlign="center" multipleRows="true">
+            <lightning:layoutItem size="12" largeDeviceSize="8" mediumDeviceSize="10" smallDeviceSize="12">
+
+                <!-- Header for Batch Template Only -->
+                <aura:if isTrue="{!v.templateMetadata.labels.sObjectNameNoNamespace == 'Batch_Template__c'}">
+                    <aura:if isTrue="{!not(v.isReadOnly)}">
+                        <div class="slds-text-align_center slds-m-vertical_medium">
+                            <h2 class="slds-text-heading_medium">{!v.templateMetadata.pageHeader}</h2>
+                        </div>
+                        <aura:set attribute="else">
+                            <div class="slds-section slds-is-open slds-p-bottom_small">
+                                <h3 class="slds-section__title slds-theme_shade">
+                                    <span class="slds-truncate slds-p-horizontal_small"
+                                          title="{!$Label.c.bgeBatchTemplateOverview}">
+                                            {!$Label.c.bgeBatchTemplateOverview}
+                                    </span>
+                                </h3>
+                            </div>
+                        </aura:set>
+                    </aura:if>
+
+                    <!--Progress Indicator-->
+                    <aura:if isTrue="{!not(v.isReadOnly)}">
+                        <div class="slds-p-bottom_large">
+                            <lightning:progressIndicator currentStep="{!v.templateMetadata.progressIndicatorStep}" type="base"
```

Comment:
Are there cases where hasError should be true?

Reply:
Good catch! I'll look at adding the error in today if it's a small add, otherwise will make a minor story to wire up the error here and in the footer.

Reply:
Was able to add this in, and it looks pretty nice!